### ISSUE TITLE
Brave Browser: Group Order/Name Bugs

### ISSUE THREAD

**Issue by WHiZSTA** [mattorchard/grouper#1]
I've been using your extention for a while now, and love how easy it is to use & optimized it is. Thanks for your work on it.

A couple bugs i've noticed though:
Occasionally when sorting, the groups created will not have names. 
Also sometimes when sorting they won't be placed in proper order.

![image](https://github.com/mattorchard/grouper/assets/18521070/3671c04e-4fe6-4fbf-b119-e07511cb9ff4)

**Comment by mattorchard** [comment]
Ooh that is interesting, I will see if I can recreate those issues.

It looks like you're using Brave (as opposed to Chrome), is that correct?

Also do you use any other extensions that can create tab groups that might influence this?

**Comment by WHiZSTA** [comment]
Yeah brave. I've tried every grouper out there, each has their own pros/cons. but yours is the one i keep coming back to and use all the time. i have no other ones loaded.

**Comment by WHiZSTA** [comment]
![image](https://github.com/mattorchard/grouper/assets/18521070/31ccddcd-4589-4ee7-a402-6f0d8eaf7e9b)
Still doing this on the live version. I've tried regrouping about 6 times so far with the same results.
